lib/models/io/attachment.dart

```diff
@@ -223,7 +223,7 @@ class Attachment {
 
   String get directory => "$baseDirectory/$guid";
 
-  String get path => "$directory/$transferName";
+  String get path => Platform.isWindows ? "$directory/$transferName".replaceAll(RegExp(r'[<>:"/\|?*]'), "_") : "$directory/$transferName";
```

Comment:
- This changes the file structure, which I assume you didn't intend to do.
- Make sure the original path separators are maintained. (i.e. you should only run replace on the `transferName`, or we'll end up with a flat file structure.)
- We should also filter out forward slashes in filenames on Linux for completeness's sake.

Reply:
Hello, I fixed the issues. Thanks